src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INCREASE_FACTOR = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size = 0;
+    private Node<K, V>[] table;
+    private int defaultCapacityThreshold;
+
+    private static class Node<K, V> {
```

Comment:
All inner classes should be at the end of class
```suggestion
    private class Node<K, V> {
```